__tests__/app.test.js

```diff
@@ -0,0 +1,33 @@
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data");
+
+beforeEach(() => seed(testData));
+afterAll(() => db.end());
+
+describe("GET /api/topics", () => {
+  test("200: responds with an array of topic objects", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        expect(Array.isArray(body.topics)).toBe(true);
+        expect(body.topics.length).toBeGreaterThan(0);
+        body.topics.forEach((each) => {
+          expect(each).toHaveProperty("slug");
+          expect(each).toHaveProperty("description");
+        });
```

Comment:
Testing that the length is greater than 0 is okay, however, knowing the exact length that you are checking against to make for more robust testing. You don't have to change this but something to consider 🤔